IPs/RSKIP177.md

```diff
@@ -0,0 +1,41 @@
+# Merged Mining Capabilities Extension
+
+|RSKIP          |177           |
+| :------------ |:-------------|
+|**Title**      |Merged Mining Capabilities Extension |
+|**Created**    |APR-2020 |
+|**Author**     |MP |
+|**Purpose**    |Sca |
+|**Layer**      |Node |
+|**Complexity** |1 |
+|**Status**     |Adopted |
+
+## Abstract
+
+The present RSKIP introduces an extension of merged mining capabilities. It allows summarized data to be hashed as part of the merged mining hash of the block. For this to be possible, network consensus must be changed.
+
```

Comment:
"summarized data" is not clear. I suggest  "a short array of data" or "an additional hash digest" or something like that.